pyproject.toml

```diff
@@ -62,9 +62,21 @@ dev = [
     "build>=1.0.3",
     "litellm>=1.60.3; sys_platform == 'win32'",
     "litellm[proxy]>=1.60.3; sys_platform != 'win32'",
-    "mcp>=1.5.0; python_version >= '3.10'",
 ]
 mcp = ["mcp; python_version >= '3.10'"]
+full_test = [
```

Comment:
Sorry this one is on me, maybe we should make this as extras to the `dev` group? 

So something like:

```
extended_test = [
  "mcp; python_version >= '3.10'",
  "pandas",
  "datasets",
]
```

Then in the action we do `uv sync -p .venv --extra dev --extra extended_test`?

Reply:
I added a group called `test_extras` for adding all extras